extensions/amp-analytics/0.1/events.js

```diff
@@ -410,6 +525,149 @@ export class IniLoadTracker extends EventTracker {
 }
 
 
+/**
+ * Timer event handler.
+ */
+class TimerEventHandler {
+  /**
+   * @param {JsonObject} timerSpec The timer specification.
+   * @param {function(): UnlistenDef=} startBuilder Factory for building start
```

Comment:
Are these optional? If so, they should start with `opt_`

Reply:
Sure. I wasn't sure what the official AMP stance on this was, since the codebase tends to contain both. I've fixed the rest of this file for consistency.